telemetry/log.go

```diff
@@ -36,7 +36,7 @@ func ZerologMiddleware(baseLogger zerolog.Logger, fluentbitClient fluentbit.Flue
 			},
 		}
 
-		logFmt := `{"time":"${time_rfc3339}","request_id":"${request_id}","remote_ip":"${remote_ip}",` +
+		logFmt := `{"time":"${time_rfc3339}","x-request-id":"${request_id}","remote_ip":"${remote_ip}",` +
```

Comment:
@jay-dee7 , could you make it '_' instead of '-' since everything else is underscored